### DISCUSSION TITLE
Include extensions from url

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#6461] [Feature Requests]
Hi everyone,

Basicly my idea is to be able to include extensions from a certain url.
So for example:
 - interfaces
 - displays

This would basicly mean that you can center some of those on netifly or somewhere else.
If there is an update, it will update everywhere at once.

**Comment by nickrum** [comment]
Interesting idea. With the changes in #6377, we would have to load those extensions on the API side to rewrite imports to be able to share e.g. Vue with the main Directus App, but that would certainly be possible.

The problem with this is that Directus instances wouldn't be fully reproducible as they would depend on an external factor that could change at any time. This could of course be considered a feature, but there are also security concerns arising from this.

With the upcoming support for npm extensions it will be possible to add extensions in any form npm allows. This also includes URLs to tarballs containing node packages. The added benefit is that those packages will be installed once and only update if the user explicitly desires so.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
